test_regress/t/t_cover_else_points.pl

```diff
@@ -0,0 +1,40 @@
+#!/usr/bin/env perl
+if (!$::Driver) { use FindBin; exec("$FindBin::Bin/bootstrap.pl", @ARGV, $0); die; }
+# DESCRIPTION: Verilator: Verilog Test driver/expect definition
+#
+# Copyright 2003-2009 by Wilson Snyder. This program is free software; you
+# can redistribute it and/or modify it under the terms of either the GNU
+# Lesser General Public License Version 3 or the Perl Artistic License
+# Version 2.0.
+# SPDX-License-Identifier: LGPL-3.0-only OR Artistic-2.0
+
+scenarios(simulator => 1);
+
+top_filename("t/t_cover_else_points.sv");
+golden_filename("t/t_cover_else_points.out");
+
```

Comment:
```suggestion
```

use the defaults and rename your file to .v instead of .sv.  Then this test passes for me.

Reply:
Ubuntu 20.04 gcc tests are still failing despite these changes.